sources/core-sw/src/filtering/qplc_unpack_prle.c

```diff
@@ -123,7 +123,7 @@ OWN_QPLC_FUN(qplc_status_t, qplc_unpack_prle_8u, (uint8_t * *pp_src,
     uint8_t  *src_ptr      = (uint8_t *) *pp_src;
     uint8_t  *src_stop_ptr = src_ptr + src_length;
     uint8_t  *dst_stop_ptr = dst_ptr + dst_length;
-    uint32_t status        = QPLC_STS_OK;
+    qplc_status_t status        = QPLC_STS_OK;
```

Comment:
Align the `=` to keep previous formatting?

Reply:
that brings up a question we had: is there a clang_format or indent configuration that is preferred for this repository?

Reply:
good question! clang_format is currently in the works, so for now for any changes we're just trying to be consistent with the code around

Reply:
whitespace is the bot’s nemesis so i will push a manual update to this bit of formatting. will be up by Thursday morning.

Reply:
updated now